From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Fri, 25 Mar 2022 20:51:03 +0100
Subject: [ARCHEOLOGY] fix emmc nano-pc-t4 (#3557)

> X-Git-Archeology: > recovered message: > * fix emmc nano-pc-t4
> X-Git-Archeology: > recovered message: > * fix node and del not used patch
> X-Git-Archeology: - Revision 03ef96715ed7fd348268506c5098800f7f64f228: https://github.com/armbian/build/commit/03ef96715ed7fd348268506c5098800f7f64f228
> X-Git-Archeology:   Date: Fri, 25 Mar 2022 20:51:03 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: fix emmc nano-pc-t4 (#3557)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0afe24c95729044910e0b3f84dc5500bcdc6524c: https://github.com/armbian/build/commit/0afe24c95729044910e0b3f84dc5500bcdc6524c
> X-Git-Archeology:   Date: Sun, 24 Apr 2022 22:33:47 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media edge to 5.17 (#3704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c: https://github.com/armbian/build/commit/2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c
> X-Git-Archeology:   Date: Mon, 04 Jul 2022 16:18:31 +0300
> X-Git-Archeology:   From: balbes150 <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 5.18 and edge 5.19
> X-Git-Archeology:
> X-Git-Archeology: - Revision 738c1613073bfd504aebc223a0d110cc6afefbe5: https://github.com/armbian/build/commit/738c1613073bfd504aebc223a0d110cc6afefbe5
> X-Git-Archeology:   Date: Sat, 12 Nov 2022 10:11:22 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media-current to 5.19.17 (#4406)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 276f346ee9c21cdcd627e54676334f4d5a9292b6: https://github.com/armbian/build/commit/276f346ee9c21cdcd627e54676334f4d5a9292b6
> X-Git-Archeology:   Date: Wed, 14 Dec 2022 10:17:09 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 6.0 and edge 6.1 (#4566)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts | 15 ++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts b/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts
index aa3857535455..a283df47e261 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts
@@ -109,6 +109,21 @@ &saradc {
 	status = "okay";
 };
 
+&uart1 {
+  status = "disabled";
+};
+
+&uart2 {
+    status = "okay";
+
+    bluetooth {
+        compatible = "realtek,rtl8723ds-bt";
+        enable-gpios = <&gpio1 RK_PD0 GPIO_ACTIVE_HIGH>;
+        host-wake-gpios = <&gpio1 RK_PD2 GPIO_ACTIVE_HIGH>;
+    };
+};
+
+
 &usb20_host_drv {
 	rockchip,pins = <0 RK_PA0 RK_FUNC_GPIO &pcfg_pull_up>;
 };
-- 
Armbian

